frappe/model/delete_doc.py

```diff
@@ -197,9 +197,18 @@ def check_if_doc_is_linked(doc, method="Delete"):
 					# raise exception only if
 					# linked to an non-cancelled doc when deleting
 					# or linked to a submitted doc when cancelling
-					frappe.throw(_('Cannot delete or cancel because {0} <a href="#Form/{0}/{1}">{1}</a> is linked with {2} <a href="#Form/{2}/{3}">{3}</a>')
-						.format(doc.doctype, doc.name, linked_doctype,
-						item.parent or item.name), frappe.LinkExistsError)
+					doc_link = '<a href="#Form/{0}/{1}">{1}</a>'.format(doc.doctype, doc.name)
+					referance_link = '<a href="#Form/{0}/{1}">{1}</a>'.format(linked_doctype, item.parent or item.name)
```

Comment:
`reference` spelling

Reply:
Also you can commonify the message, (include doctype as part of the text), make an inner function and call it.